app/routes/_index.tsx

```diff
@@ -35,19 +53,31 @@ export default function Index() {
             typografi={TypografiTyper.StegHeadingH1}
           />
           <Språkvelger />
+
           <VeilederHilsen
             innhold={tekster.veilederhilsenInnhold}
             hilsen={tekster.brukerHilsen}
           />
           <SamtykkePanel
+            tittel={tekster.samtykkePanelTittel}
             innhold={tekster.samtykkePanelMelding}
             samtykke={tekster.samtykkePanelSamtykke}
             feilmelding={tekster.samtykkePanelFeilmelding}
+            onSamtykkeEndring={håndtereSamtykkeEndring}
+            feilmeldingAktivert={feilmeldingAktivert}
           />
+          <Button
+            variant={samtykkeErBekreftet ? 'primary' : 'secondary'}
+            onClick={
+              samtykkeErBekreftet
+                ? () => navigate(nestePath)
+                : håndtereKnappeTrykk
+            }
```

Comment:
Burde vi flytte inn logikken for sjekk om samtykkeErBekreftet inn i selve håndterKnappeTrykk-funksjonen? Fordelen er at selve <Button>-komponenten blir lettere å lese

Reply:
Lager et nytt kort på det, så kan vi se på det med alle endringene fra branchen om samtykkePanel